lib/msal-angular/package.json

```diff
@@ -21,7 +21,7 @@
     "build:all:prod": "npm run build:browser && npm run build:prod",
     "build:browser": "cd ../.. && npm run build --workspace=@azure/msal-common --workspace=@azure/msal-browser",
     "build:pack": "npm run build && cd dist && npm pack",
-    "deploy": "npm run build:all:prod && npm publish ./dist",
+    "deploy": "npm run build:all:prod && npm publish ./dist --workspaces=false",
     "deploy:beta": "npm run build:all:prod && npm publish ./dist --tag beta",
```

Comment:
should we remove this one while we're at it?

Reply:
Done, thanks!